activerecord/test/cases/adapters/mysql2/nested_deadlock_test.rb

```diff
@@ -78,6 +78,57 @@ class Sample < ActiveRecord::Base
       assert_predicate connection, :active?
     end
 
+    test "rollback exception is swallowed after a rollback" do
```

Comment:
Is this specific to mysql or should we also include a test in test/cases/adapters/postgresql/transaction_nested_test.rb for postgres?

Reply:
At this point applies to `mysql2` adapter only

But I would put my answer as "it only applies to adapters that have `savepoint_errors_invalidate_transactions` configured as `true`", which, by implication, scopes the change to mysql only

https://github.com/rails/rails/blob/682353e3dea9bead97c5fe83b799f65723954412/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L78-L80